docs/docs/how-to/images-and-media/using-gatsby-plugin-image.md

```diff
@@ -0,0 +1,201 @@
+```shell
+npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-transformer-sharp
+```
+
+You then need to add the plugins to your `gatsby-config.js`:
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    `gatsby-plugin-image`,
+    `gatsby-plugin-sharp`,
+    `gatsby-transformer-sharp`,
+  ],
+}
+```
+
+If you already have some of these plugins installed, please check that they're updated to the latest version.
+
+<!-- TODO: add exact minimum version when we reach GA -->
+
+## Using the Gatsby Image components
+
+### Decide which component to use
+
+The Gatsby Image plugin includes two image components: one for static and one for dynamic images. The simplest way to decide which you need to is to ask yourself: _"will this image be the same every time the component or template is used?"_. If it will always be the same, then use `StaticImage`. If it will change, whether through data coming from a CMS or different values passed to a component each time you use it, then it is a dynamic image and you should use the `GatsbyImage` component.
+
+### Static images
+
+If you are using an image that will be the same each time the component is used, such as a logo or front page hero image, you can use the `StaticImage` component. The image can be either a local file in your project, or an image hosted on a remote server. Any remote images are downloaded and resized at build time.
+
+1. **Add the image to your project.**
+
+   If you are using a local image, copy it into the project. A folder such as `src/images` is a good choice.
+
+2. **Add the `StaticImage` component to your template.**
+
+   Import the component, then set the `src` prop to point to the image you added earlier. The path is relative to the source file itself. If your component file was `src/components/dino.js`, then you would load the image like this:
+
+   ```jsx:title=src/components/dino.js
+   import { StaticImage } from "gatsby-plugin-image"
+
+   export function Dino() {
+     return <StaticImage src="../images/dino.png" alt="A dinosaur" />
+   }
+   ```
+
+   If you are using a remote image, pass the image URL in the `src` prop:
+
+   ```jsx:title=src/components/kitten.js
+   import { StaticImage } from "gatsby-plugin-image"
+
+   export function Kitten() {
+     return <StaticImage src="https://placekitten.com/800/600" alt="A kitten" />
+   }
+   ```
+
+   When you build your site, the `StaticImage` component will load the image from your filesystem or from the remote URL, and it will generate all the sizes and formats that you need to support a responsive image.
+
+   Because the image is loaded at build time, you cannot pass the filename in as a prop, or otherwise generate it outside of the component. It should either be a static string, or a local variable in the component's scope.
+
+   > **Important:** Remote images are downloaded and resized at build time. If the image is changed on the other server, it will not be updated on your site until you rebuild.
```

Comment:
```suggestion
**Important:** Remote images are downloaded and resized at build time. If the image is changed on the other server, it will not be updated on your site until you rebuild.
```
If I recall correctly using callouts isn't great for accessibility if it isn't necessary.

Reply:
Nice tip, thanks! TIL